apps/updatenotification/tests/BackgroundJob/ResetTokenTest.php

```diff
@@ -50,29 +55,53 @@ public function testRunWithNotExpiredToken(): void {
 		$this->config
 			->expects($this->never())
 			->method('deleteSystemValue');
+		$this->appConfig
+			->expects($this->never())
+			->method('deleteKey');
+		$this->logger
+			->expects($this->never())
+			->method('warning');
+		$this->logger
+			->expects($this->once())
+			->method('debug');
 
-		static::invokePrivate($this->resetTokenBackgroundJob, 'run', [null]);
+		$this->invokePrivate($this->resetTokenBackgroundJob, 'run', [null]);
```

Comment:
```suggestion
		static::invokePrivate($this->resetTokenBackgroundJob, 'run', [null]);
```

Static is the correct/new stuff. `$this->invokePrivate` actually generates a warning in IDEs?

Ref https://github.com/nextcloud/server/commit/ffb3a3e33a2b0468a39ab9f11cf8f63f020bd0bb